tests/testthat/helper-rng.R

```diff
@@ -30,25 +31,25 @@ hypergeometric_r <- function(random_real) {
     a <- lfactorial(m) +
       lfactorial(n1 - m) +
       lfactorial(k - m) +
-      lfactorial((n2 - k)  + m)
+      lfactorial((n2 - k) + m)
 
-    numerator <- (n - k) * k * n1  * n2
+    numerator <- (n - k) * k * n1 * n2
     denominator <- (n - 1) * n * n
     d <- floor(1.5 * sqrt(numerator / denominator)) + 0.5
 
     x_l <- m - d + 0.5
     x_r <- m + d + 0.5
 
     k_l <- exp(a -
-               lfactorial(x_l) -
-               lfactorial(n1 - x_l) -
-               lfactorial(k - x_l) -
-               lfactorial((n2 - k)  + x_l))
+                 lfactorial(x_l) -
```

Comment:
We'll get churn here if one of us doesn't reconfigure our indent rules (looks like throughout). No idea which of the two versions is really prevailing style. Not wildly fussed though